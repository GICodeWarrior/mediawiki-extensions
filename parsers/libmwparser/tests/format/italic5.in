a'''b''c'''d'''

 a'''b''c'''d'''

aa'''b''c'''d'''

 ''''b''c'''d'''

''''b''c'''d'''

a'''b'' '''d'''

a'''b'' ''' d'''