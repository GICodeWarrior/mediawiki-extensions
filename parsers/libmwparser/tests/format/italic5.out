BEGIN ARTICLE
    BEGIN PARAGRAPH
        WORD[a]
        SPECIAL[']
        BEGIN ITALIC
            WORD[b]
        END ITALIC
        WORD[c]
        BEGIN BOLD
            WORD[d]
        END BOLD
    END PARAGRAPH
    BEGIN PRE
        WORD[a]
        SPECIAL[']
        BEGIN ITALIC
            WORD[b]
        END ITALIC
        WORD[c]
        BEGIN BOLD
            WORD[d]
        END BOLD
    END PRE
    BEGIN PARAGRAPH
        WORD[aa]
        SPECIAL[']
        BEGIN ITALIC
            WORD[b]
        END ITALIC
        WORD[c]
        BEGIN BOLD
            WORD[d]
        END BOLD
    END PARAGRAPH
    BEGIN PRE
        SPECIAL[']
        SPECIAL[']
        BEGIN ITALIC
            WORD[b]
        END ITALIC
        WORD[c]
        BEGIN BOLD
            WORD[d]
        END BOLD
    END PRE
    BEGIN PARAGRAPH
        SPECIAL[']
        SPECIAL[']
        BEGIN ITALIC
            WORD[b]
        END ITALIC
        WORD[c]
        BEGIN BOLD
            WORD[d]
        END BOLD
    END PARAGRAPH
    BEGIN PARAGRAPH
        WORD[a]
        SPECIAL[']
        BEGIN ITALIC
            WORD[b]
        END ITALIC
        SPACE[ ]
        BEGIN BOLD
            WORD[d]
        END BOLD
    END PARAGRAPH
    BEGIN PARAGRAPH
        WORD[a]
        BEGIN BOLD
            WORD[b]
            BEGIN ITALIC
                SPACE[ ]
            END ITALIC
        END BOLD
        BEGIN ITALIC
            SPACE[ ]
            WORD[d]
            SPECIAL[']
        END ITALIC
    END PARAGRAPH
END ARTICLE
